[
  {"LiveFrom": "01/01/2018", "CaseTypeID": "PROBATE_ExceptionRecord", "ID": "createException", "Name": "Create an exception record", "Description": "Create an exception record", "DisplayOrder": 1, "PostConditionState": "ScannedRecordReceived", "SecurityClassification": "Public"},
  {"LiveFrom": "01/01/2018", "CaseTypeID": "PROBATE_ExceptionRecord", "ID": "attachToExistingCase", "Name": "Attach record to existing case", "Description": "Attach record to existing case", "DisplayOrder": 2, "PreConditionState(s)": "ScannedRecordReceived", "PostConditionState": "ScannedRecordAttachedToCase", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/attach_case", "RetriesTimeoutURLAboutToSubmitEvent": 10, "SecurityClassification": "Public", "ShowEventNotes": "N", "CanSaveDraft": "N"},
  {"LiveFrom": "01/01/2018", "CaseTypeID": "PROBATE_ExceptionRecord", "ID": "rejectRecord", "Name": "Reject record", "Description": "Reject record", "DisplayOrder": 3, "PreConditionState(s)": "ScannedRecordReceived", "PostConditionState": "ScannedRecordRejected", "SecurityClassification": "Public", "ShowEventNotes": "N", "CanSaveDraft": "N"},
  {"LiveFrom": "01/01/2018", "CaseTypeID": "PROBATE_ExceptionRecord", "ID": "updateManually", "Name": "Manually handle record", "Description": "Manually handle record", "DisplayOrder": 4, "PreConditionState(s)": "ScannedRecordReceived", "PostConditionState": "ScannedRecordManuallyHandled", "SecurityClassification": "Public", "ShowEventNotes": "N", "CanSaveDraft": "N"}
]
